test/hotreload-test.lua

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env tarantool
+
+local tap = require('tap')
+local ffi = require('ffi')
+local key_def_lib = require('tuple.keydef')
+
+-- Presence of the methods confirms that correct metatype is set
+-- for the <struct tuple_merge_source> ctype.
```

Comment:
Sorry for nitpicking, but presence of module functions (BTW, it is not quite correct to call them methods) just verifies that they're here. While presence of instance methods proves that the metatype is correct.

Reply:
Sorry for delay. Yes, thanks for notice. This comment is related to function below. I moved to correct place and added appropriate comment for this function.